lab-justin/lib/task-crud.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Task = require('../model/task');
```

Comment:
It's usually a good practice to put all of your requirements for npm packages at the top, then all the requirements for other files that you wrote.